debian/changelog

```diff
@@ -1,9 +1,16 @@
-cloud-init (25.3-~1gxxxxxxxx-0ubuntu1) UNRELEASED; urgency=medium
+cloud-init (25.3~1g890873f5-0ubuntu1) questing; urgency=medium
```

Comment:
As there is not yet a 25.3 upstream version, should not this be `25.2~1g890873f5-0ubuntu1`?

Reply:
@aciba90 the 25.3~1 makes it sort lower (diminished version) than that eventual official release of 25.3
```
dpkg --compare-versions 25.3~1 lt 25.3 && echo LESS THAN || echo BROKEN
LESS THAN
```